assets/app/view/game/corporation.rb

```diff
@@ -303,7 +312,13 @@ def render_shares
 
       def render_revenue_history
         last_run = @corporation.operating_history[@corporation.operating_history.keys.max].revenue
-        h('td.bold', "Last Run: #{@game.format_currency(last_run)}")
+        h('div.bold', "Last Run: #{@game.format_currency(last_run)}")
+      end
+
+      def render_operating_order(n)
+        round = @game.round
```

Comment:
don't need to store this var you only use it once

Reply:
twice (was three times, hence it’s there). Where’s your cutoff? ;)

Reply:
ah didn't see you used it twice that's fine